IHP/RouterSupport.hs

```diff
@@ -286,8 +287,19 @@ parseFuncs parseIdType = [
                     |> fromASCIIBytes
                     |> \case
                         Just uuid -> uuid |> unsafeCoerce |> Right
-                        Nothing -> Left BadType { field = "", value = Just queryValue, expectedType = "UUID" }
+                        Nothing ->
+                            -- We couldn't parse the UUID, so try Maybe (Id record),
+                            -- where we have a @Just@ prefix before the UUID.
+                            queryValue
```

Comment:
@mpscholten Is there a way to debug such a function and have it print the value of `queryValue` ?

Reply:
Well, I've just fixed it `"Just" ` not "Just "`. But still would be nice to know if there's a way to debug

Reply:
@mpscholten It's indeed a tricky problem. Is there a way to debug/print the values in those functions?

Reply:
Check out https://hackage.haskell.org/package/base-4.18.0.0/docs/Debug-Trace.html#v:trace